docs/developer/app-store/apps/adyen.mdx

```diff
@@ -53,6 +53,51 @@ If Adyen surpasses this limit, the App will return a FAILURE status with an appr
 
 For Adyen to appear as [available payment gateway](../../checkout/finalizing#listing-available-payment-gateways), you need to [install it in the Saleor Dashboard](../overview#usage). You must obtain the API key from Adyen and paste it into the Adyen App configuration form. Then, a wizard will guide you through the process of configuring the Adyen App, setting up the webhook to receive notifications from Adyen, generating the HMAC key, and adding allowed origins for the Client Key that's used on your Storefront.
 
+### Configuring Adyen
```

Comment:
I think this section is "installing adyen"

and section below will be `### configuring adyen`